cupy_setup_build.py

```diff
@@ -464,7 +470,8 @@ def preconfigure_modules(compiler, settings):
             # Fail on per-library condition check (version requirements etc.)
             installed = True
             errmsg = ['The library is installed but not supported.']
-        elif module['name'] in ('thrust', 'cub') and nvcc_path is None:
+        elif (module['name'] in ('thrust', 'cub')
+                and (nvcc_path is None and hipcc_path is None)):
             installed = True
             errmsg = ['nvcc command could not be found in PATH.',
```

Comment:
let's change the error message too, if people is installing this without cuda, an error about nvcc can be confusing.